.github/workflows/release-desktop-mas.yml

```
@@ -75,6 +75,37 @@ jobs:
         run: |
           cd ./apps/desktop && yarn add electron-builder@23.6.0
 
+      - name: Update package.json for MAS build
+        run: |
+          cd ./apps/desktop
+          # Add workspaces and installConfig to package.json
+          node -e '
+            const fs = require("fs");
+            const pkg = JSON.parse(fs.readFileSync("package.json", "utf8"));
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)